### PR TITLE
Paraswap dex add more blockchain

### PR DESCRIPTION
**Pull request by springzh**
Brief comments on the purpose of your changes:

This PR is to add 5 new blockchains data for ParaSwap. 
Also include some minor tweak to previous created ParaSwap spells.

**For Dune Engine V2**

I've checked that:

### General checks:
* [x] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [x] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [x] if adding a new model, I added a test
* [x] the filename is unique and ends with .sql
* [x] each sql file is a select statement and has only one view, table or function defined  
* [x] column names are `lowercase_snake_cased`
* [x] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [x] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [ ] `coin_id` represents the ID of the coin on coinpaprika.com
* [ ] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [x] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [x] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [x] where block_time >= date_trunc("day", now() - interval '1 week')
* [x] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [x] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @springzh 

There are some trades with 0 as`evt_index`, as paraswap did not emit some events to save gas.
https://dune.com/queries/2455736?d=1

**Comment by springzh** [comment]
From Mounir (CEO of ParaSwap):
"Also, some functions in Ethereum don’t emit events (for gas optimisation)".

For reference, here are some sample transactions that don't emit their own events:
-- swap_type, block_number, tx_hash
-- erc20 to erc20, 13302850, 0x34777d43e267d17862375293bb2e3cc4e8bd44abe7dca761891690c290eebee0
-- eth to erc20, 13512018, 0xf196c5393f402cef9cbe01341d55ab90d2233b650b4cb1381dffb21f2ff6bb73
-- erc20 to eth, 13300892, 0x99d61733a0e8f4bf5d1fdf25e4929de987ac883e4b70c81f01af4a53ca3d800f

The "Swap" or similar events we can find in the above transactions are not emitted from ParaSwap contract. They are from other contracts such as Uniswap.

**Review by jeff-dude — CHANGES_REQUESTED** [review]
minor feedback below, but great stuff here!

File: models/paraswap/ethereum/paraswap_v4_ethereum_trades.sql [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+            t.block_number,
+            t.block_time,
+            tx.`from` AS user_address,
+            '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2' AS tokenIn, -- WETH
+            sum(case
+                when t.`from` = tx.`from` then cast(t.value AS decimal(38, 0))
+                else -1 * cast(t.value AS decimal(38, 0))
+            end) AS amountIn,
+            MAX(t.trace_address) AS trace_address,
+            CAST(0 AS bigint) AS evt_index
```
[the remark this answers is not in the document]
Comment (springzh) [reply models/paraswap/ethereum/paraswap_v4_ethereum_trades.sql]: Done. Updated to -1.

File: models/paraswap/ethereum/paraswap_v4_ethereum_trades.sql [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+        UNION ALL
+        
+        SELECT t.tx_hash,
+            t.block_number,
+            t.block_time,
+            t.`to` AS user_address,
+            '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2' AS tokenOut, -- WETH
+            cast(t.value AS decimal(38, 0)) AS amountOut,
+            t.trace_address,
+            CAST(0 AS bigint) AS evt_index
```
[the remark this answers is not in the document]
Comment (springzh) [reply models/paraswap/ethereum/paraswap_v4_ethereum_trades.sql]: Done. Updated to -1.

File: models/paraswap/ethereum/paraswap_v5_ethereum_trades.sql [hunk 3]
```diff
@@ -0,0 +1,420 @@
...
+            t.block_number,
+            t.block_time,
+            tx.`from` AS user_address,
+            '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2' AS tokenIn, -- WETH
+            sum(case
+                when t.`from` = tx.`from` then cast(t.value AS decimal(38, 0))
+                else -1 * cast(t.value AS decimal(38, 0))
+            end) AS amountIn,
+            MAX(t.trace_address) AS trace_address,
+            CAST(0 AS bigint) AS evt_index
```
[the remark this answers is not in the document]
Comment (springzh) [reply models/paraswap/ethereum/paraswap_v5_ethereum_trades.sql]: Done. Updated to -1.

File: models/paraswap/ethereum/paraswap_v5_ethereum_trades.sql [hunk 4]
```diff
@@ -0,0 +1,420 @@
...
+        UNION ALL
+        
+        SELECT t.tx_hash,
+            t.block_number,
+            t.block_time,
+            t.`to` AS user_address,
+            '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2' AS tokenOut, -- WETH
+            cast(t.value AS decimal(38, 0)) AS amountOut,
+            t.trace_address,
+            CAST(0 AS bigint) AS evt_index
```
[the remark this answers is not in the document]
Comment (springzh) [reply models/paraswap/ethereum/paraswap_v5_ethereum_trades.sql]: Done. Updated to -1.

**Review by jeff-dude — APPROVED** [review]
thanks all

**Pull request merged**